cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1606,7 +1557,7 @@ public void setSubscriptionTopics(String[] topicIds, CompletionListener completi
                             topicsArray.put(topicId);
                         }
 
-                        jsonBody.put("channelId", this.channelId);
+                        jsonBody.put("channelId", getChannelId(getContext()));
```

Comment:
we will need it to mock the channel id from test cases and it will be the same as this.channelId because we are saving channel id to local preferences every time we do init with a different channel id